funfact/lang/interpreter/_initialization.py

```diff
@@ -1,12 +1,29 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-import numpy as np
+import jax.numpy as np
+import jax
+#from jax import random
+#import numpy as np
 from ._base import TranscribeInterpreter
 
 
+
+class JaxRng:
+    
+    def __init__(self, key=0):
+        self.key = jax.random.PRNGKey(key)
+        
+    def rand(self, size, dtype=np.float32):
```

Comment:
I'd suggest we call this method `normal` to be consistent with the underlying JAX method.